x/stake/handler_sidechain.go

```diff
@@ -105,6 +105,10 @@ func handleMsgEditSideChainValidator(ctx sdk.Context, msg MsgEditSideChainValida
 		validator.SideFeeAddr = msg.SideFeeAddr
 	}
 
+	if len(msg.SideConsAddr) != 0 && sdk.IsUpgrade(sdk.BEP159) {
```

Comment:
Please add a case tha:
During the upgrade of BEP159,  assume few validators have already upgraded to latest version, most are sill old binary, we use a new cli to send a `MsgEditSideChainValidator` to a new validator, which try to change the SideConsAddr,  I guess it will cause consensus issue, because the old validator an not even decode the message.

Reply:
Since the field is `omitempty`, I'll check the field in `ValidateBasic` function, return error if it's not empty before BEP159.

```
	SideConsAddr []byte `json:"side_cons_addr,omitempty"`
```

Reply:
fixed